ci/test/05_before_script.sh

```diff
@@ -32,6 +32,9 @@ if [ -n "$ANDROID_TOOLS_URL" ]; then
   DOCKER_EXEC unzip -o "$ANDROID_TOOLS_PATH" -d "${ANDROID_HOME}/cmdline-tools"
   DOCKER_EXEC "yes | ${ANDROID_HOME}/cmdline-tools/tools/bin/sdkmanager --install \"build-tools;${ANDROID_BUILD_TOOLS_VERSION}\" \"platform-tools\" \"platforms;android-${ANDROID_API_LEVEL}\" \"ndk;${ANDROID_NDK_VERSION}\""
 
+  # Remove cached Qt
+  DOCKER_EXEC rm -rf "${DEPENDS_DIR}"/built/aarch64-linux-android/qt
```

Comment:
Is there any insight available as to why this fixes the ci?

Reply:
There's partial insight.

APK CI build fails when there is no `qtbase*.tar.xz` archive in `depends/sources` to extract "Qt for Android" java files from.
Line above forces a redownload of Qt sources and fixes that particular issue. 

I could produce a proper fix if I knew why or when the `sources` are removed in between CI runs. I see a `clean-all` target that wipes `$(SOURCES_PATH)` clean but surely that's not it? Since that one would delete `build` as well.

Reply:
depends/sources is never cached and never has been

See #21541 for a fix

Reply:
That was extermely fast. Thank you!

Closing this.